server/db/games.js

```diff
@@ -1,42 +1,46 @@
-export const games = [
-  {
-    id: "1",
-    title: "E.T.",
-    platform: "Atari",
-    rating: 1.0,
-    reviews: ["101", "301"],
-    theme: "",
-  },
-  {
-    id: "2",
-    title: "Super Mario Bros.",
-    platform: "Nintendo(nes)",
-    rating: 5.0,
-    reviews: [],
-    theme: "1",
-  },
-  {
-    id: "3",
-    title: "Ninja Gaiden",
-    platform: "Nintendo(nes)",
-    rating: 5.0,
-    reviews: ["201"],
-    theme: "1",
-  },
-  {
-    id: "4",
-    title: "Resident Evil VII",
-    platform: "Playstation 4",
-    rating: 3.5,
-    reviews: [],
-    theme: "2",
-  },
-  {
-    id: "5",
-    title: "Eternal Darkness",
-    platform: "Gamecube",
-    rating: 4.0,
-    reviews: [],
-    theme: "2",
-  },
-];
+export const games = [{
```

Comment:
PR looks good.  One thing I did notice your formatter seemed to format this entire file.  Will you add `"prettier.requireConfig": true,` to your `settings.json` in vscode.  That way we are all on the default stylings.  Other than that looks awesome.  I'll test it in the playground tonight.